DfE.FindInformationAcademiesTrusts/Pages/Trusts/Academies/PupilNumbers.cshtml

```diff
@@ -0,0 +1,44 @@
+@* Add the route to use hyphenation between words *@
+@page "/trusts/academies/pupil-numbers"
+@model PupilNumbersModel
+
+@{
+  Layout = "Academies/_AcademiesLayout";
+}
+
+<section class="govuk-!-margin-top-4 govuk-!-margin-bottom-9 app-table-container">
+  <table class="govuk-table" data-module="moj-sortable-table" aria-describedby="academies-pupil-numbers-link">
+    <thead class="govuk-table__head">
+      <tr class="govuk-table__row">
+        <th scope="col" class="govuk-body govuk-table__header" aria-sort="ascending">School name</th>
+        <th scope="col" class="govuk-body govuk-table__header" aria-sort="none">Phase and age range</th>
+        <th scope="col" class="govuk-body govuk-table__header" aria-sort="none">Pupil numbers</th>
+        <th scope="col" class="govuk-body govuk-table__header" aria-sort="none">Pupil capacity</th>
+        <th scope="col" class="govuk-body govuk-table__header" aria-sort="none"> % full</th>
+      </tr>
+    </thead>
+    <tbody class="govuk-table__body">
+      @foreach (var academy in Model.Trust.Academies)
+      {
+        <tr class="govuk-table__row" data-testid="academy-row">
+          <th scope="row" class="govuk-body govuk-table__header" data-sort-value="@academy.EstablishmentName">
+            @academy.EstablishmentName<br/><span class="govuk-!-font-weight-regular">URN: @academy.Urn</span>
+          </th>
+          <td class="govuk-body govuk-table__cell" data-sort-value="@Model.PhaseAndAgeRangeSortValue(academy)">
+            @academy.PhaseOfEducation<br/>
+            @academy.AgeRange.Minimum - @academy.AgeRange.Maximum
+          </td>
+
+          <td class="govuk-body govuk-table__cell" data-sort-value="@academy.NumberOfPupils">@academy.NumberOfPupils?.ToString("N0")</td>
+          <td class="govuk-body govuk-table__cell" data-sort-value="@academy.SchoolCapacity">@academy.SchoolCapacity?.ToString("N0")</td>
+          <td class="govuk-body govuk-table__cell" data-sort-value="@academy.PercentageFull">
+            @if (academy.PercentageFull is not null)
+            {
+              @Html.Raw(academy.PercentageFull?.ToString("N0") + "%")
```

Comment:
Do we definitely need this `@Html.Raw`?
Will `@academy.PercentageFull?.ToString("N0")%` work?

Reply:
It will, but it doesn't know how to handle the %, so I will either need to wrap that bit in a html tag or a helper, e.g.

`@if (academy.PercentageFull is not null)
            {
              @academy.PercentageFull?.ToString("N0")<span>%</span>
            }`

Reply:
Have pushed this alternative, can you take a look?